roseau/load_flow/models/buses.py

```diff
@@ -279,6 +279,28 @@ def get_connected_buses(self) -> Iterator[Id]:
                 to_add = set(element._connected_elements).difference(visited)
                 remaining.update(to_add)
 
+    def res_voltage_unbalance(self) -> float:
+        """Calculate the voltage unbalance on this bus according to the IEC definition.
+
+        Voltage Unbalance Factor:
+
+        :math:`VUF = \\frac{|V_n|}{|V_p|} * 100 (\\%)`
+
+        Where :math:`V_n` is the negative-sequence voltage and :math:`V_p` is the positive-sequence
+        voltage.
+        """
+        # https://std.iec.ch/terms/terms.nsf/3385f156e728849bc1256e8c00278ad2/771c5188e62fade5c125793a0043f2a5?OpenDocument
+        if self.phases not in {"abc", "abcn"}:
+            msg = f"Voltage unbalance is only available for 3-phases buses, bus {self.id!r} has phases {self.phases!r}"
+            logger.error(msg)
+            raise RoseauLoadFlowException(msg, code=RoseauLoadFlowExceptionCode.BAD_PHASE)
+        # We use the potentials here which is equivalent to using the "line to neutral" voltages as
+        # defined by the standard. The standard also has this note:
+        # NOTE 1 Phase-to-phase voltages may also be used instead of line to neutral voltages.
+        potentials = self._res_potentials_getter(warning=True)
+        _, vp, vn = phasor_to_sym(potentials[:3])  # (0, +, -)
+        return abs(vn) / abs(vp) * 100
```

Comment:
In the `TransformerParameter` class, for the  `__init__` method, the arguments `i0` and `vsc` are percentages too. They are provided as values between 0 and 1 (eg. "0.1" for "10%"). I think we should do the same. The unit `""` is used with `pint` to avoid confusion. If you prefer returning percentages (between 0 and 100), please use the `"percent"` unit of pint.

Reply:
Mmmm. On one hand the standars defines VUF as a value between 0 and 100, on the other hand I like to be consistent in the code. It is a tough call.